doc/Language/quoting.pod6

```diff
@@ -174,38 +174,26 @@ My favorite color is blue!
 =end code
 
 X<|\ (quoting)>
+
 The C<qq> form – usually written using double quotes – allows for
-interpolation of backslash sequences and variables, i.e., variables can be
-written within the string so that the content of the variable is inserted into
-the string. It is also possible to escape variables within a C<qq>-quoted
-string:
+interpolation of backslash escape sequences (like C<q:backslash>), all sigiled
+variables (like C<q:scalar:array:hash:function>), and any code inside C<{...}>
+(like C<q:closure>).
```

Comment:
I was a bit confused here, but then I understood that what you want to say is that `qq` works in the same way as q with a series of adverbs. It might be clear withing the whole context of the page, but either a link or a fleshed out clarification might be clearer.